resource-usage-collect-addon/helm/test.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+set -e
+
+# Install the chart with test values
+echo "Installing the chart with test values..."
+helm install resource-usage-collect-addon . \
+  --namespace open-cluster-management-addon \
+  --create-namespace \
+  -f test-values.yaml
+
+# Wait for resources to be created
+echo "Waiting for resources to be created..."
+sleep 10
+
+# Run the tests
+echo "Running the tests..."
+helm test resource-usage-collect-addon -n open-cluster-management-addon
+
+# Uninstall the chart
+echo "Uninstalling the chart..."
+helm uninstall resource-usage-collect-addon -n open-cluster-management-addon
+
+echo "Tests completed successfully!" 
```

Comment:
Do you have an example of how the helm test is used in other chart? I don't see a chart has test.sh, tests/* files inside chart before. Maybe we need remove these to make the chart look more standardized.